tests/testthat/_snaps/demo/demo.md

```diff
@@ -571,7 +571,7 @@ cat(news, sep = "\n")
 ## 
 ## # tea 0.0.1
 ## 
-## - Same as previous version.
+## - Internal changes only.
```

Comment:
@krmlr Seeing what happens in the demo: actually this behavior should probably depend on `which`?

- `which=dev` ok to force a version with internal changes only.
- `which=patch` or more, in that case one probably wants the header to change, the grouping to be done? (#97)

Reply:
should there even be `bump_version()` for something else than dev?

Reply:
isn't it what you were saying earlier actually? :sweat_smile:

Reply:
Can you please rephrase the question?

Reply:
In a perfect world/repo do you agree that one
- would run bump_version() often, only `which=dev` (if not dev, use `pre_release()`)
- would run `pre_release()` without the need to select a which (guessed from the changelog if conventional commits syntax is used, potentially with confirmation)

And in `pre_release()` one should not get "no changes". One should, at this point, re-group all items from the intermediary development versions.

Reply:
in the demo currently one runs `bump_version()` for a sort of release which is where I realized seeing "no changes since last version" in the changelog was weird.

Reply:
Agree 💯 . We keep the parts composable though, so `bump_version("non-dev")` remains supported?

Reply:
why should it remain supported? for current users' workflows? I tend to think it might be confusing. (users with alternative workflows could use `bump_version_impl()`, after all :innocent: )

Reply:
Should we add `bump_dev_version()` ?

Reply:
I don't think so, it's a longer name. It's easier to remember to run `bump_version()` regularly (although you might argue some people will not do it as they can use GHA).

Reply:
`bump_version()` and `bump_release_version()` ?

Reply:
Let's stay with `bump_version()` for now and perhaps avoid commenting too much on the `"which"` argument in the vignette. We can deprecate at some point, I prefer separating new features from breaking changes and removing features if possible.